public/app/features/alerting/unified/components/contact-points/NewMessageTemplate.tsx

```diff
@@ -1,16 +1,16 @@
-import { EntityNotFound } from 'app/core/components/PageNotFound/EntityNotFound';
-
 import { useAlertmanager } from '../../state/AlertmanagerContext';
+import { withPageErrorBoundary } from '../../withPageErrorBoundary';
+import { AlertmanagerPageWrapper } from '../AlertingPageWrapper';
 import { TemplateForm } from '../receivers/TemplateForm';
 
-const NewMessageTemplate = () => {
+function NewMessageTemplate() {
   const { selectedAlertmanager } = useAlertmanager();
 
-  if (!selectedAlertmanager) {
```

Comment:
why do we change this logic? do you prefer delegating the error to the template form?